modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/Commenting/Default.cshtml

```diff
@@ -32,6 +32,7 @@
           style="@(string.IsNullOrEmpty(repliedCommentId?.ToString() ?? "") ? "" : "display:none")">
         <form class="cms-comment-form">
             <input hidden value="@(repliedCommentId?.ToString() ?? "")" name="repliedCommentId" />
+            <input hidden asp-for="Input.AllowExternalUrls" name="allowExternalUrls" />
```

Comment:
This doesn't seem to be safe enough, but i understand the reason behind it. You want to set AllowExternalUrls separately for each comment section. However, i don't think it is neccessary. You can set it in options by `EntityType`. Example code:

```
        Configure<CmsKitCommentOptions>(options =>
        {
            options.AllowedExternalUrls = new Dictionary<string, List<string>>
            {
                {
                    "Product",
                    new List<string>()
                    {
                        "https://abp.io/"
                    }
                }
            };
        });
```

With this implementation, you can check it on backend. And i think this will cover the real need. Comments on `Product` EntityTypes won't allow the external urls globally. You can ask @ebicoglu about this.

Reply:
Yes, this makes more sense. I will update it according to your comment.